zer0-os/zOS
# feat: conversation creation with inviter on registration completion

### What does this do?
- fetches the `inviter` data from the registration and creates a conversation between the two users.

### Why are we making this change?
- following the behaviour when using the sendbird client, the changes are to automatically connect new users with their inviters by creating a conversation when completing registration and logging into the app.

### How do I test this?
- get an invite code from your current user > go to the `/get-access` route and enter the code > complete registration > when logged in, check a conversation has been created and appears in your conversation list with the inviter.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?

Join Demo


https://github.com/zer0-os/zOS/assets/39112648/a5c5392f-fa96-4aa7-9959-7ca79fd5de41
